Example/src/ScrollViewOffsetCallScrollToExample.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import Animated, {
+  useScrollViewOffset,
+  useAnimatedRef,
+  withTiming,
+} from 'react-native-reanimated';
+import { Button, StyleSheet, Text, View } from 'react-native';
+
+export default function ScrollViewOffsetCallScrollToExample() {
+  const aref = useAnimatedRef<Animated.ScrollView>();
+  const scrollHandler = useScrollViewOffset(aref);
+
+  const onChangeScrollValue = () => {
+    scrollHandler.value = Math.random() * 5000;
+  };
+
+  const onAnimatedChangeScrollValue = () => {
+    scrollHandler.value = withTiming(Math.random() * 5000);
+  };
+
+  return (
+    <>
+      <View style={styles.positionView}>
+        <Text>Test</Text>
+        <Button
+          title="Scroll without animation"
+          onPress={onChangeScrollValue}
+        />
+        <Button
+          title="Scroll with animation"
+          onPress={onAnimatedChangeScrollValue}
+        />
+      </View>
+      <View style={styles.divider} />
+      <Animated.ScrollView
+        ref={aref}
+        scrollEventThrottle={16}
```

Comment:
is it necessary for the throttle parameter to be present? Typically with nativedriver / reanimated you want throttle to be set to 1 because all events are processed on the UI thread anyways